library/packages/src/lib/y2packager/license.rb

```diff
@@ -87,18 +95,19 @@ def cache
     #
     # Bear in mind that `fetcher` will be ignored if `content` is specified.
     #
-    # @param fetcher [:libzypp] Fetcher to retrieve licenses information. For the time
-    #   being, only :libzypp is supported.
-    # @param content [String]   License content. If this argument is given, this
-    #   string is used as the license's content (and `source` is ignored).
-    def initialize(fetcher: nil, content: nil)
+    # @param product_name [String] Product name to retrieve license information
+    # @param content      [String] License content. If this argument is given, this
+    #   string is used as the license's content (and `product_name` is ignored).
```

Comment:
this is a bit confusing for me as from code it looks like it is used only for DEFAULT_LANG, otherwise it is fetched, not?

Reply:
Yes, you are right. I will update the documentation to something like

```ruby

@param content [String] License content. If given, this string will be used as the license's content for the default languge, ignoring the `product_name`.
```
Sounds better for you?